addons/survey/security/survey_security.xml

```diff
@@ -66,24 +57,15 @@
             <field name="perm_read" eval="1"/>
             <field name="perm_create" eval="1"/>
         </record>
-        <record id="survey_question_rule_survey_user_read" model="ir.rule">
-            <field name="name">Survey question: officer: read all</field>
-            <field name="model_id" ref="survey.model_survey_question"/>
-            <field name="domain_force">[(1, '=', 1)]</field>
-            <field name="groups" eval="[(4, ref('group_survey_user'))]"/>
-            <field name="perm_unlink" eval="0"/>
-            <field name="perm_write" eval="0"/>
-            <field name="perm_read" eval="1"/>
-            <field name="perm_create" eval="0"/>
-        </record>
-        <record id="survey_question_rule_survey_user_cw" model="ir.rule">
-            <field name="name">Survey question: officer: create/write/unlink linked to own survey only</field>
+
+        <record id="survey_question_rule_survey_user" model="ir.rule">
+            <field name="name">Survey question: officer: survey restricted to self or no one, or standalone</field>
```

Comment:
What do you mean by "standalone" ?

Reply:
Side question: seems question does not have required "survey_id" (hence 'standalone' I suppose), is it intended ?